trackers/huggingface/graphs.py

```diff
@@ -613,3 +626,165 @@ def results_table(df: pd.DataFrame):
+                    "value": sk_val["Optimize ONNX file"],
+                    "itemStyle": {"color": "white", "borderColor": "white"},
+                },
+                {
+                    "name": "Converts to FP16",
+                    "value": sk_val["Converts to FP16"],
+                    "itemStyle": {"color": "white", "borderColor": "white"},
+                },
+                {
+                    "name": "Builds Nvidia",
+                    "value": sk_val["Builds Nvidia"],
+                    "itemStyle": {
+                        "color": device_colors["nvidia"],
+                        "borderColor": device_colors["nvidia"],
+                    },
+                },
+                {
+                    "name": "Builds Groq",
+                    "value": sk_val["Builds Groq"],
+                    "itemStyle": {
+                        "color": device_colors["groq"],
+                        "borderColor": device_colors["groq"],
+                    },
+                },
+                {
+                    "name": "Builds x86",
+                    "value": sk_val["Builds x86"],
+                    "itemStyle": {
+                        "color": device_colors["x86"],
+                        "borderColor": device_colors["x86"],
+                    },
+                },
+            ],
+            "label": {
+                "position": "insideTopLeft",
+                "borderWidth": 0,
+                "fontSize": 16,
+                "color": "white",
+                "textBorderWidth": 0,
+                "formatter": "{c}",
+            },
+            "links": [
+                {
+                    "source": "All models",
+                    "target": "Convert to ONNX",
+                    "value": summ.all_models,
+                },
+                {
+                    "source": "Convert to ONNX",
+                    "target": "Optimize ONNX file",
+                    "value": summ.optimized_onnx,
+                },
+                {
+                    "source": "Optimize ONNX file",
+                    "target": "Converts to FP16",
+                    "value": summ.fp16_onnx,
+                },
+                {
+                    "source": "Converts to FP16",
+                    "target": "Builds Nvidia",
+                    "value": int(
+                        summ.nvidia
+                        * summ.fp16_onnx
+                        / (summ.x86 + summ.nvidia + summ.groq)
+                    ),
+                },
+                {
+                    "source": "Converts to FP16",
+                    "target": "Builds Groq",
+                    "value": int(
+                        summ.groq
+                        * summ.fp16_onnx
+                        / (summ.x86 + summ.nvidia + summ.groq)
+                    ),
+                },
+                {
+                    "source": "Converts to FP16",
+                    "target": "Builds x86",
```

Comment:
More "build" terminology to potentially change

Reply:
All solved